recipes/george/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "george" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "da69a2c11ef2e1496f492d6055b83d7f180d22fb89479667facfb9b77a56a0d8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/dfm/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
Why skip windows? It looks like `eigen` and the other dependencies are all available for that platform. I don't see anything in the upstream source repo that would indicate that it should be excluded.